packages/guides/src/RenderContext.php

```diff
@@ -30,6 +31,8 @@ class RenderContext
     private string $destinationPath;
 
     private DocumentNode $document;
+    /** @var DocumentNode[] */
+    private array $allDocuments;
```

Comment:
Would it be an idea to collect all documents in the projectnode and pass that in the renderingcontext?

Reply:
This does not work because someone :tm: wants the nodes to be immutable